Files/Views/LayoutModes/GenericFileBrowser.xaml.cs

```diff
@@ -102,6 +107,11 @@ public GenericFileBrowser()
             AllView.PointerCaptureLost += AllView_ItemPress;
         }
 
+        protected override void InitializeCommandsViewModel()
+        {
+            CommandsViewModel = new BaseLayoutCommandsViewModel(new BaseLayoutCommandImplementationModel(ParentShellPageInstance));
```

Comment:
Do you need to spin up a new view model for every instance?

Reply:
@yaichenbaum that's what was also done with Interaction case. IShellPage with Interaction was populated for every instance. It's just that's more clear in this example.

Reply:
We could experiment with new solutions to this after the `Interaction` is completely removed. It's some pretty exciting performance benefits stuff that wasn't possible with `Interaction` being in IShellPage :eyes: